.github/workflows/main.yml

```
@@ -693,3 +692,96 @@
         PR_REF_SHA: ${{ github.event.pull_request.head.sha }}
       if: contains(steps.files.outputs.modified, 'configure.ac') || contains(steps.files.outputs.modified, 'shell/install.sh') || contains(steps.files.outputs.all, 'src_ext') || contains(steps.files.outputs.all, '.github/workflows')
       run: bash -exu .github/scripts/main/hygiene.sh
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)